x-pack/plugins/fleet/common/types/models/epm.ts

```diff
@@ -231,7 +231,7 @@ export interface RegistryElasticsearch {
   'index_template.mappings'?: object;
 }
 
-export type RegistryVarType = 'integer' | 'bool' | 'password' | 'text' | 'yaml';
+export type RegistryVarType = 'integer' | 'bool' | 'password' | 'text' | 'yaml' | 'string';
```

Comment:
sorry I am late to the official review, but I am surprised at this change. AFAIK there is no var type called `string` on package spec side, only `text`? https://github.com/elastic/package-spec/blob/master/versions/1/data_stream/manifest.spec.yml#L50-L55

Reply:
Good question. Right now packages use both, e.g. `apm` here has a lot of `string`s: https://github.com/elastic/package-storage/blob/snapshot/packages/apm/0.1.0-dev.7/manifest.yml#L44-L136

Reply:
Opened https://github.com/elastic/package-storage/issues/1011